src/Spectre.AlgorithmsCli.Tests/Spectre.AlgorithmsCli.Tests.vcxproj

```diff
@@ -60,9 +60,10 @@
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <AdditionalOptions>/std:c++latest %(AdditionalOptions)</AdditionalOptions>
       <TreatWarningAsError>true</TreatWarningAsError>
+      <AdditionalIncludeDirectories>$(SolutionDir)Spectre.libDivik;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Do we need this having `$(SolutionDir)` in `Additional Include Directories` through Property Sheet?